vocode/streaming/telephony/client/vonage_client.py

```diff
@@ -0,0 +1,72 @@
+from typing import Optional
+from vocode.streaming.models.telephony import VonageConfig
+from vocode.streaming.telephony.client.base_telephony_client import BaseTelephonyClient
+import vonage
+
+from vocode.streaming.telephony.constants import VONAGE_CONTENT_TYPE
+
+
+class VonageClient(BaseTelephonyClient):
+    def __init__(self, base_url, vonage_config: VonageConfig):
+        super().__init__(base_url)
+        self.vonage_config = vonage_config
+        self.client = vonage.Client(
+            key=vonage_config.api_key,
+            secret=vonage_config.api_secret,
+            application_id=vonage_config.application_id,
+            private_key=vonage_config.private_key,
+        )
+        self.voice = vonage.Voice(self.client)
+
+    def get_telephony_config(self):
+        return self.vonage_config
+
+    # TODO(EPD-186): support recording in Vonage
+    def create_call(
+        self,
+        conversation_id: str,
+        to_phone: str,
+        from_phone: str,
+        record: bool = False,
+        digits: Optional[str] = None,
+    ) -> str:  # identifier of the call on the telephony provider
+        response = self.voice.create_call(
+            {
+                "to": [{"type": "phone", "number": to_phone, "dtmfAnswer": digits}],
```

Comment:
will this cause issues when digits is None?

Reply:
nope! i've been exclusively trying it with `digits = None`